src/components/Modals.js

```diff
@@ -110,14 +109,14 @@ class GetSshPassphraseModal extends Component {
       });
   }
 
-  isValidPassphrase = () => {
+  isValidPassphrase() {
     if (this.state.passphrase) {
       return true;
     }
     return false;
   }
 
-  renderErrorMessage = () => {
+  renderErrorMessag() {
```

Comment:
This incorrectly renamed the function (missing trailing 'e')

Reply:
Good catch.